schemas/tlc/1.3.0/sxl.yaml

```diff
@@ -114,46 +114,68 @@ objects:
       S0001:
         description: |-
           Signal group status.
-          Provides the status of each signal group, including basic information such as green, yellow and red. But also detailed technical information.
-          Can be used to draw a live signal group diagram as well provide diagnostic information about the performance of the controller.
+          Provides the status of each signal group, e.g. green, yellow, red,
+          or other states used in the signal program.
+          See the section on signal group states for more information about
+          the possible states.
+
+          Can be used to draw a live signal group diagram as well as to provide
+          diagnostic information about the performance of the controller.
+
+          The base cycle counter is based on just the internal clock and the
+          cycle length. Controllers with synchronized clocks and the same cycle
+          length will therefore have identical base cycle counters.
+
+          The cycle clock is based on the base cycle counter, but shifts the
+          phase using the offset set in the controller:
+
+          c = (b + o) modulo t
+
+          where:
+          c = cycle counter
+          b = base cycle counter
+          o = offset
+          t = cycle length
```

Comment:
I've noticed that you change "cycle time" to "cycle length". However, S0028 is called "cycle time" so isn't more consistent to stick with "cycle time"? It's not a big deal, since either one is acceptable.

Reply:
yes let's stick with cycle time

Reply:
fixed